source/sixda.f90

```diff
@@ -788,7 +788,7 @@ subroutine runda
           write(7,*) '5'
```

Comment:
`Write(7,*)` ? Not using the allocation, using format * ?

Reply:
Got nothing to do with this PR.